modules/nodes-cluster-enabling-features-about.adoc

```diff
@@ -17,10 +17,8 @@ The following Technology Preview features are enabled by this feature set:
 ** Microsoft Azure File CSI Driver Operator. Enables the provisioning of persistent volumes (PVs) by using the Container Storage Interface (CSI) driver for Microsoft Azure File Storage.
 ** CSI automatic migration. Enables automatic migration for supported in-tree volume plug-ins to their equivalent Container Storage Interface (CSI) drivers. Supported for:
 *** Amazon Web Services (AWS) Elastic Block Storage (EBS)
-*** OpenStack Cinder
-*** Azure Disk
+*** Google Compute Engine Persistent Disk
 *** Azure File
-*** Google Cloud Platform Persistent Disk (CSI)
```

Comment:
Azure File is GA in 4.11: 
The changes in CSI migration Tech Preview seems correct.

However, it seems like we need to remove Azure File CSI Driver Operator from Tech Preview because it is GA in 4.11: https://github.com/openshift/api/pull/1155


		with("CSIMigrationAWS").             // sig-storage, jsafrane, Kubernetes feature gate
		with("CSIMigrationGCE").             // sig-storage, fbertina, Kubernetes feature gate
		with("CSIMigrationAzureFile").       // sig-storage, fbertina, Kubernetes feature gate
		with("CSIMigrationvSphere").         // sig-storage, fbertina, Kubernetes feature gate